tests/agent-plugin-unit/test_mk_oracle.sh

```diff
@@ -99,6 +100,7 @@ SYNC_ASM_SECTIONS=instance
 ASYNC_ASM_SECTIONS=asm_diskgroup
 CACHE_MAXAGE=300
 OLRLOC=/other/path
+MAX_TASKS=1
```

Comment:
Please use another value != default_value, eg. MAX_TASKS=5
Also add egde cases for MAX_TASKS<=0